src/ComputeSharp.D2D1/Extensions/RectangleExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Drawing;
+using System.Runtime.CompilerServices;
+
+namespace ComputeSharp.D2D1;
+
+/// <summary>
+/// Helper methods for <see cref="Rectangle"/>.
+/// </summary>
+public static class RectangleExtensions
+{
+    /// <summary>
+    /// Modifies a target <see cref="Rectangle"/> instance to be a D2D1 infinite rectangle.
+    /// </summary>
+    /// <param name="rectangle">The target <see cref="Rectangle"/> instance to modify.</param>
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    public static void ToD2D1Infinite(this ref Rectangle rectangle)
```

Comment:
recommend using the verb `Make` instead of `To`, following the pattern used elsewhere in System.Drawing e.g. https://docs.microsoft.com/en-us/dotnet/api/system.drawing.region.makeinfinite?view=dotnet-plat-ext-6.0